mantle/build

```diff
@@ -43,7 +50,7 @@ cross_build() {
 		echo "Building $a/$1"
 		mkdir -p "bin/$a"
 		CGO_ENABLED=0 GOARCH=$a \
-			go build -mod=vendor -ldflags "${ldflags}" \
+			go build -mod=vendor -ldflags "${ldflags}" ${race} \
```

Comment:
Shouldn't `-race` be conditional on `$a` for `cross_build` instead of on `uname -m`?

Reply:
Good catch, I'll push a follow on commit shortly.

Reply:
Okay, fixed.